src/notation/view/widgets/selectdialog.cpp

```diff
@@ -47,18 +48,18 @@ SelectDialog::SelectDialog(QWidget* parent)
     setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
     m_element = contextItem(globalContext()->currentNotation()->interaction());
-    type->setText(qtrc("elementName", m_element->typeUserName()));
+    type->setText(qtrc("notation", m_element->typeUserName()));
 
     switch (m_element->type()) {
     case ElementType::ACCIDENTAL:
-        subtype->setText(qApp->translate("accidental", m_element->subtypeName().toUtf8()));
+        subtype->setText(qtrc("notation", m_element->subtypeName()));
         break;
     case ElementType::SLUR_SEGMENT:
-        subtype->setText(qApp->translate("elementName", m_element->subtypeName().toUtf8()));
+        subtype->setText(qtrc("notation", m_element->subtypeName()));
         break;
     case ElementType::FINGERING:
     case ElementType::STAFF_TEXT:
-        subtype->setText(qApp->translate("TextStyle", m_element->subtypeName().toUtf8()));
+        subtype->setText(qtrc("notation", m_element->subtypeName()));
```

Comment:
I'm afraid these new translation contexts are wrong. They need to be the same as where the string is defined. For example, the case of `m_element->typeUserName()`, it must be `"elementName"`, because the string is defined in that context, see EngravingObject::typeUserName().

Reply:
I agree.